### ISSUE TITLE
Incredibly Large Losses Being Reported during Training 

### ISSUE THREAD

**Issue by cmbowyer13** [tensorflow/models#4537]
Please go to Stack Overflow for help and support:

http://stackoverflow.com/questions/tagged/tensorflow

Also, please understand that many of the models included in this repository are experimental and research-style code. If you open a GitHub issue, here is our policy:

1. It must be a bug, a feature request, or a significant problem with documentation (for small docs fixes please send a PR instead).
2. The form below must be filled out.

**Here's why we have that policy**: TensorFlow developers respond to issues. We want to focus on work that benefits the whole community, e.g., fixing bugs and adding features. Support only helps individuals. GitHub also notifies thousands of people when issues are filed. We want them to see you communicating an interesting problem, rather than being redirected to Stack Overflow.

------------------------

### System information
- **What is the top-level directory of the model you are using**:models/object_detection/Object-Detection/ I am working from an earlier commit. 
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: Yes. 
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Linux
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: TF 1.5
- **Bazel version (if compiling from source)**:
- **CUDA/cuDNN version**: 
- **GPU model and memory**:
- **Exact command to reproduce**:

You can collect some of this information using our environment capture script:

https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh

You can obtain the TensorFlow version with

python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

### Describe the problem
Describe the problem clearly here. Be sure to convey here why it's a bug in TensorFlow or a feature request. I recently after a much painful debugging period of several weeks finally got a tensorflow model set up for training on images of shape [None, None, 1]. One issue i had was i was decoding using tf.uint8 before, and now i'm decoding raw data using tf.int32 and the training started smoothly, but this is what it's reporting so far :
018-06-14 10:12:46.832715: I tensorflow/core/platform/cpu_feature_guard.cc:137] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2
INFO:tensorflow:Restoring parameters from faster_rcnn_resnet101_kitti_2018_01_28_edit/model.ckpt
INFO:tensorflow:Starting Session.
INFO:tensorflow:Saving checkpoint to path training_dir/Training8_FRCNN/model.ckpt
INFO:tensorflow:Starting Queues.
INFO:tensorflow:global_step/sec: 0
INFO:tensorflow:Recording summary at step 0.
INFO:tensorflow:global step 1: loss = 2111620.5000 (92.193 sec/step)
INFO:tensorflow:global_step/sec: 0.00882505
INFO:tensorflow:Recording summary at step 1.
INFO:tensorflow:global step 2: loss = 3.0828 (64.799 sec/step)
INFO:tensorflow:global step 3: loss = 2.8934 (55.309 sec/step)
INFO:tensorflow:global_step/sec: 0.0166667
INFO:tensorflow:Recording summary at step 3.
INFO:tensorflow:global step 4: loss = 397766.8750 (62.280 sec/step)
INFO:tensorflow:global step 5: loss = 151397.8594 (56.366 sec/step)
INFO:tensorflow:global_step/sec: 0.0166667
INFO:tensorflow:Recording summary at step 5.
INFO:tensorflow:global step 6: loss = 0.9747 (62.037 sec/step)
INFO:tensorflow:global step 7: loss = 1310138.8750 (52.413 sec/step)
INFO:tensorflow:global_step/sec: 0.0166666
INFO:tensorflow:Recording summary at step 7.
INFO:tensorflow:global step 8: loss = 60647.3984 (58.173 sec/step)
INFO:tensorflow:global step 9: loss = 299056.2188 (49.699 sec/step)
INFO:tensorflow:Saving checkpoint to path training_dir/Training8_FRCNN/model.ckpt
INFO:tensorflow:global step 10: loss = 0.6832 (47.258 sec/step)
INFO:tensorflow:global_step/sec: 0.024997
INFO:tensorflow:Recording summary at step 10.
INFO:tensorflow:global step 11: loss = 1.0021 (50.228 sec/step)
INFO:tensorflow:global step 12: loss = 41494812.0000 (41.387 sec/step)
INFO:tensorflow:global_step/sec: 0.0166688
INFO:tensorflow:Recording summary at step 12.
INFO:tensorflow:global step 13: loss = 1.0045 (46.058 sec/step)
INFO:tensorflow:global step 14: loss = 1.0183 (37.177 sec/step)
INFO:tensorflow:global step 15: loss = 4732048.0000 (37.373 sec/step)
INFO:tensorflow:global_step/sec: 0.025
INFO:tensorflow:Recording summary at step 15.
INFO:tensorflow:global step 16: loss = 158327.9375 (43.464 sec/step)

Why and how is the Loss growing so large, an then shrinking back down near 1.0?

### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.
